style.css

```diff
@@ -0,0 +1,536 @@
+.header {
+  display: flex;
+  flex-direction: column;
+}
+
+.top-nav {
+  background-color: black;
+  height: auto;
+}
+
+.small-nav__menu {
+  display: flex;
+  justify-content: flex-end;
+  padding-right: 10px;
+  align-items: center;
+  margin-right: 25px;
+}
```

Comment:
- [ ] I think it would be better to add a [universal selector](https://developer.mozilla.org/en-US/docs/Web/CSS/Universal_selectors) ```*``` in your CSS file to match elements of any type and set rules to avoid repetition in your codes, 👍 
   - Currently, you have used ```font-family: 'Lato', sans-serif;``` **10** times which is not a good practice, this can be grabbed in the universal selector  😎
   - Your may also set the default ```margin and padding to 0``` to remove the default added spacing between our elements on the page 👍 

### Example
```
* {
   margin: 0;
   padding: 0;
   font-family: 'Lato', sans-serif;
}
```

---

This space around your page will be removed✨😉
![image](https://user-images.githubusercontent.com/102757126/217872393-25afaad5-f858-43a9-a275-314950932cc4.png)